frontend/src/budgetSimulation/budgetSimulation.js

```diff
@@ -33,16 +33,23 @@ class Citizen extends React.Component {
     render() {
         const statistic = (
             <Popover id={"popover-basic"}>
-                <Popover.Title as={"h3"}> {this.props.data.name} </Popover.Title>
                 <Popover.Content>
-                    {Object.keys(this.props.data.traits).map((trait, key) =>
-                        (<div key={key}>
-                            <strong>
-                                {trait.split("_").join(" ")}: &nbsp;
-                            </strong>
-                            {this.props.data.traits[trait] ? "True" : "False"}
-                            <br/>
-                        </div>)
+                    Citizen {this.props.data.name} &nbsp;
+                    {Object.keys(this.props.data.traits).map((trait, i) => {
+                        console.log(Object.keys(this.props.data.traits).length);
+                        return (<>
+                            {this.props.data.traits[trait]
+                                ? trait.split("_").join(" ")
+                                : trait.split("_")
+                                    .join(" ")
+                                    .replace("has", "does not have")
+                                    .replace("is", "is not")
+                            }{i === Object.keys(this.props.data.traits).length - 2
```

Comment:
plz no nested ternary! super hard to read.

Reply:
no more nested ternary